index.html

```diff
@@ -146,97 +163,99 @@ <h2 class="header tonic__header" >Multi-Post
-            <div class="frame">
-              <div class="skills__lang-header">
-                <div class="skills__lang-text">Frameworks</div>
-                <div class="skills__lang-icon-wrapper">
-                  <object class="skills__lang-icon" data="/img/SVG/chevron-small-right.svg" height="20" width="20"></object>
-                </div>
-              </div>
+            <ul class="skills__lang-list">
+              <li class="skills__lang-list--item"><a href="#" class="skills__lang-list--link">
+                <span class="skills__lang-list--iconjs"></span>
+                <span class="skills__lang-list--text">Javascript</span>
+              </a></li>
+              <li class="skills__lang-list--item"><a href="#" class="skills__lang-list--link">
+                <span class="skills__lang-list--iconht"></span>
+                <span class="skills__lang-list--text">HTML</span>
+                </a></li>
+              <li class="skills__lang-list--item"><a href="#" class="skills__lang-list--link">
+                <span class="skills__lang-list--iconcs"></span>
+                <span class="skills__lang-list--text">CSS</span>
+              </a></li>
+            </ul>
           </div>
-          </li>
-          <li class="skills__list-item">
-            <div class="skill">
-              <div class="skills__lang-header">
-                <div class="skills__lang-text">Languages</div>
-                <div class="skills__lang-icon-wrapper">
-                  <object class="skills__lang-icon" data="/img/SVG/chevron-small-right.svg" height="20" width="20"></object>
-                </div>
+        </li>
+        <li class="skills__list-item">
+          <div class="frame">
+            <div class="skills__lang-header">
+              <div class="skills__lang-text">Frameworks</div>
+              <div class="skills__lang-icon-wrapper">
+                <object class="skills__lang-icon" data="/img/SVG/chevron-small-right.svg" height="20" width="20"></object>
               </div>
             </div>
-          </li>
-        </ul>
-      </div>
+        </div>
+        </li>
+        <li class="skills__list-item">
+          <div class="skill">
+            <div class="skills__lang-header">
+              <div class="skills__lang-text">Languages</div>
+              <div class="skills__lang-icon-wrapper">
+                <object class="skills__lang-icon" data="/img/SVG/chevron-small-right.svg" height="20" width="20"></object>
+              </div>
+            </div>
+          </div>
+        </li>
+      </ul>
     </div>
   </div>
-  <!-- end of works -->
+  <!-- end -->
 
   <!-- Contact Form  -->
-  <footer>
+  <footer id="contact-id">
     <div class="contact">
       <h2 class="contact__header">Contact me</h2>
     <p class="contact__text">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
```

Comment:
- [ ] [REQUIRED] Please, try to increase the width of the form a bit to match the origianl design

Your Design | Figma Design
------------|----------------
![image](https://user-images.githubusercontent.com/60979926/165879844-5c6850da-02ca-47cd-9f51-885489441e2d.png) | ![image](https://user-images.githubusercontent.com/60979926/165879906-6354d0cf-d240-4684-aca6-0e2b88177b77.png)